app/src/main/java/com/framgia/fsalon/screen/report/ReportViewModel.java

```diff
@@ -0,0 +1,134 @@
+package com.framgia.fsalon.screen.report;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.support.annotation.IntDef;
+import android.support.v4.app.Fragment;
+import android.view.View;
+
+import com.framgia.fsalon.BR;
+import com.framgia.fsalon.screen.report.reportsales.ReportSalesFragment;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.framgia.fsalon.screen.report.ReportViewModel.TabButton.TAB_BILLS;
+import static com.framgia.fsalon.screen.report.ReportViewModel.TabButton.TAB_CUSTOMERS;
+import static com.framgia.fsalon.screen.report.ReportViewModel.TabButton.TAB_SALES;
+
+/**
+ * Exposes the data to be used in the Report screen.
+ */
+public class ReportViewModel extends BaseObservable implements ReportContract.ViewModel {
+    private ReportContract.Presenter mPresenter;
+    private ReportPagerAdapter mAdapter;
+    private int mTabPosition;
+    private boolean mIsSale, mIsCustomer, mIsBill;
+
+    public ReportViewModel(Fragment fragment) {
+        List<Fragment> fragments = new ArrayList<>();
+        fragments.add(ReportSalesFragment.newInstance());
+        fragments.add(new Fragment());
+        fragments.add(new Fragment());
+        mAdapter = new ReportPagerAdapter(fragment.getChildFragmentManager(), fragments);
+        setAdapter(mAdapter);
+    }
+
+    @Bindable
+    public boolean isSale() {
+        return mIsSale;
+    }
+
+    public void setSale(boolean sale) {
+        mIsSale = sale;
+        notifyPropertyChanged(BR.sale);
+    }
+
+    @Bindable
+    public boolean isCustomer() {
+        return mIsCustomer;
+    }
+
+    public void setCustomer(boolean customer) {
+        mIsCustomer = customer;
+        notifyPropertyChanged(BR.customer);
+    }
+
+    @Bindable
+    public boolean isBill() {
+        return mIsBill;
+    }
+
+    public void setBill(boolean bill) {
+        mIsBill = bill;
+        notifyPropertyChanged(BR.bill);
+    }
+    /**
+     * TabButton
+     */
+    @IntDef({TAB_SALES, TAB_CUSTOMERS, TAB_BILLS})
+    @Retention(RetentionPolicy.SOURCE)
+    public @interface TabButton {
+        int TAB_SALES = 0;
+        int TAB_CUSTOMERS = 1;
+        int TAB_BILLS = 2;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.